packages/react-scripts/config/webpack.config.js

```diff
@@ -365,7 +365,7 @@ module.exports = function(webpackEnv) {
                   isEnvProduction ? 'production' : isEnvDevelopment && 'development',
                   [
                     'babel-plugin-named-asset-import',
-                    'babel-preset-react-app',
+                    'babel-preset-frontier',
```

Comment:
does this one here need to be prefixed with @fs?

Reply:
no. This is just a cache identifier. It could be any string.